swiftsimio/visualisation/volume_render.py

```diff
@@ -233,6 +246,15 @@ def render_gas_voxel_grid(
         defaults to False, but can speed up the creation of large images 
         significantly at the cost of increased memory usage.
 
+    rotation_matrix: np.array, optional
+        Rotation matrix (3x3) that describes the rotation of the box around
+        ``rotation_center``. In the default case, this provides a projection
```

Comment:
Again, same comment about `rotation_matrix` - we need a better comment describing what the rotation matrix actually does here.